src/status_im/multiaccounts/update/core.cljs

```diff
@@ -46,3 +46,12 @@
     {:db (if setting-value
            (assoc-in db [:multiaccount setting] setting-value)
            (update db :multiaccount dissoc setting))}))
+
+(fx/defn toggle-opensea-nfts-visibility
+  {:events [::toggle-opensea-nfts-visiblity]}
+  [cofx visible?]
```

Comment:
`?` is usually used for functions

Reply:
@flexsurfer where does it come from? I always used it for booleans

Reply:
tacit clojurists agreement, just to understand if its a predicate or not

Reply:
but we don't have agreement in status and we have mixed usage

Reply:
> tacit clojurists agreement

is that according to some doc or...? I haven't seen or heard it anywhere yet

Reply:
what I mean is that I haven't seen any argument against using it for booleans

Reply:
I didn't know this. I have been using `?` for both booleans and predicates: https://www.reddit.com/r/Clojure/comments/b7db39/question_marks_in_clojure/

Reply:
I personally use it for both predicates and booleans similar to Ruby community. Marking this as resolved as its not actionable.